controllers/controller_test.go

```diff
@@ -550,3 +551,20 @@ func testNoFinalizer() {
 		return snr.Finalizers, err
 	}, 10*time.Second, 250*time.Millisecond).Should(BeEmpty())
 }
+
+func eventuallyUpdateNode(updateFunc func(*v1.Node), isStatusUpdate ...bool) {
+	By("Verify that node was updated successfully")
+
+	EventuallyWithOffset(1, func() error {
+		node := &v1.Node{}
+		if err := k8sClient.Client.Get(context.TODO(), unhealthyNodeNamespacedName, node); err != nil {
+			return err
+		}
+		updateFunc(node)
+		if len(isStatusUpdate) > 0 && isStatusUpdate[0] {
```

Comment:
IMHO the code would be more simple and readable when `isStatusUpdate` is a normal bool instead of a variadic, especially because having more than one of them will never make sense